### PR TITLE
[VA template] update va template base on sample

### LINKED ISSUE #2144: [VA template] update va template base on sample
#### What happens?
update va template base on sample

### PR DESCRIPTION
**Pull request by DingmaomaoBJTU**
### Purpose
*What is the context of this pull request? Why is it being done?*
Close #2144 [VA template] update va template base on sample 

### Changes
Apply the change in va sample project to va template code in next branch.

### Tests
Manual

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

#### Bots
- [ ] I have validated that new and updated responses use appropriate [Speak](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-text-to-speech?view=azure-bot-service-3.0) and [InputHint](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-add-input-hints?view=azure-bot-service-3.0) properties to ensure a high-quality speech-first experience
- [ ] I have replicated language model changes across the English, French, Italian, German, Spanish, and Chinese `.lu` files and validated that deployment is successful

#### Deployment Scripts
- [ ] I have replicated my changes in the **Virtual Assistant Template** and **Sample** projects
- [ ] I have replicated my changes in the **Skill Template** and **Sample** projects

### CODE REVIEW

File: templates/Virtual-Assistant-Template/csharp/Template/VA/Services/SkillIntentRecognizer.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Globalization;
```
Comment (lzc850612) [inline templates/Virtual-Assistant-Template/csharp/Template/VA/Services/SkillIntentRecognizer.cs:2]: are the same changes made to the Sample of VA?

**Review by lzc850612 — CHANGES_REQUESTED** [review]

File: templates/Virtual-Assistant-Template/csharp/Template/VA/Services/SkillIntentRecognizer.cs (hunk 1 above)
Comment (DingmaomaoBJTU) [reply templates/Virtual-Assistant-Template/csharp/Template/VA/Services/SkillIntentRecognizer.cs:2]: Yes, this change is port from sample of VA.

File: templates/Virtual-Assistant-Template/csharp/Template/VA/VA.csproj [hunk 2]
```diff
@@ -23,8 +26,8 @@
     <PackageReference Include="Microsoft.Bot.Builder.Integration.ApplicationInsights.Core" Version="4.5.1" />
     <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.5.1" />
     <PackageReference Include="Microsoft.Bot.Builder.StreamingExtensions" Version="4.5.1-preview1" />
-    <PackageReference Include="Microsoft.Bot.Builder.Skills" Version="4.5.3" />
-    <PackageReference Include="Microsoft.Bot.Builder.Solutions" Version="4.5.3" />
+    <PackageReference Include="Microsoft.Bot.Builder.Skills" Version="4.6.0-daily6" />
```
Comment (darrenj) [inline templates/Virtual-Assistant-Template/csharp/Template/VA/VA.csproj:29]: Should not be using daily versions! Released versions

File: templates/Virtual-Assistant-Template/csharp/Template/VA/VA.csproj [hunk 3]
```diff
@@ -3,6 +3,9 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.2</TargetFramework>
     <NoWarn>NU1701</NoWarn>
+    <RestoreAdditionalProjectSources>
+      https://botbuilder.myget.org/F/aitemplates/api/v3/index.json;
```
Comment (darrenj) [inline templates/Virtual-Assistant-Template/csharp/Template/VA/VA.csproj:7]: Why has this been added?

**Review by darrenj — COMMENTED** [review]
Also - there is no information either in the PR or the issue as to why the PR is being performed so it can be validated. Please use the template fields!

**Review by lauren-mills — CHANGES_REQUESTED** [review]
I think these changes should be made using the template from master instead of next, since it is syncing the current template version with the sample. I think targeting next is why there are issues with packages and daily builds.

File: templates/Virtual-Assistant-Template/csharp/Template/VA/VA.csproj (hunk 2 above)
Comment (DingmaomaoBJTU) [reply templates/Virtual-Assistant-Template/csharp/Template/VA/VA.csproj:29]: This is a template in next branch, I just make it to sync up with the sample in next branch, which has to include the lib daily package.

File: templates/Virtual-Assistant-Template/csharp/Template/VA/VA.csproj (hunk 3 above)
Comment (DingmaomaoBJTU) [reply templates/Virtual-Assistant-Template/csharp/Template/VA/VA.csproj:7]: Because we use the lib daily package, so we need to include additional feeds.

**Comment by DingmaomaoBJTU** [comment]
@darrenj  @lauren-mills  This change's purpose is to sync the va template code with va sample project in next branch. Not sure whether this is the right approach, in my opinion, the master branch's va template should sync with va sample in master, and similar for next branch. So my question is, how should we maintain the va template in each branch? (Or we can just leave it there and do the sync up stuff when the code reach to master)

File: templates/Virtual-Assistant-Template/csharp/Template/VA/VA.csproj (hunk 3 above)
Comment (darrenj) [reply templates/Virtual-Assistant-Template/csharp/Template/VA/VA.csproj:7]: I'm confused - if we make changes in the next branch and then merge into master at release time these changes would flow which is not what we want? Are you after a "daily build" version of a template?

**Comment by lauren-mills** [comment]
I think for next branch it is okay to be out of sync, because that sync work will be part of the release checklist. Its better to get all the changes in to the sample then we can update the template once its stable

**Comment by DingmaomaoBJTU** [comment]
I see~~ Let's close this then ^_^~

**Pull request closed without merging**